### ISSUE TITLE
[REQUEST] calculate gradients for customized inputs

### ISSUE THREAD

**Issue by LiJiaqi96** [MisaOgura/flashtorch#43]
Thanks a lot for developing fantastic package FlashTorch!  

I have a question about the inputs. My inputs contains two parts:  
- the image
- additional information of image (one node)  
 
The additional node is inputed in the middle layers with torch.cat.  

In such case, can I use FlashTorch to obtain my saliency maps? Thanks a lot!
